a5/Dockerfile

```diff
@@ -1,6 +1,7 @@
 # rosindustrial/a5:kinetic
-FROM rosindustrial/noether-webtools:kinetic
+FROM rosindustrial/noether:kinetic
 MAINTAINER Austin.Deric@gmail.com
 RUN apt-get update && apt-get install -y \
-  ros-kinetic-stage \
-  sudo
+  ros-kinetic-stage
+RUN apt-get clean && \
+  rm -rf /var/lib/apt/lists/*
```

Comment:
apology not needed, i really appreciate the feedback!  I took care of the && on this latest commit when i built every dockerfile. We will be doing automated builds soon which trigger docker hub builds based on github commits.